src/routes/balances/balances.service.ts

```diff
@@ -30,12 +30,12 @@ export class BalancesService {
     trusted?: boolean,
     excludeSpam?: boolean,
   ): Promise<Balances> {
```

Comment:
We should also change the arguments of `getBalances` (and other functions in this service).

Reply:
[9db8374](https://github.com/safe-global/safe-client-gateway-nest/pull/610/commits/9db8374f600be9d8fbe58d5c91761aef0ea8b5cf)